### PR TITLE
landing page: final changes

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description
- updated navbar
- swap fund & login button
- custom newsletter

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)
- landing page

### CODE REVIEW

**Comment by thelostone-mc** [comment]
Nav changes

x1


Also @mbecom / @Saptak -> The news letter content doesn't seem to get updated based on the params I'm passing from `views.py`
Am I missing anything ?

**Comment by mbeacom** [comment]
This is on staging.

File: app/retail/views.py [hunk 1]
```diff
@@ -54,6 +54,9 @@ def index(request):
         'slides': slides,
         'slideDurationInMs': 6000,
         'active': 'home',
+        'hide_newsletter_caption': True,
```
Comment (mbeacom) [inline app/retail/views.py]: What's the utility of this bool? Are we altering this value at any point? 🤔

**Comment by mbeacom** [comment]
screenshot 2018-06-08 15 04 39

File: app/retail/views.py (hunk 1 above)
Comment (SaptakS) [reply app/retail/views.py]: @mbeacom I think what @thelostone-mc has done here is he is sending this boolean only to landing-page, which makes it non-hidden in all the other pages. Hence both the caption and consent are shown in every other page except the landing page.

**Pull request merged**